public/home.js

```diff
@@ -1,6 +1,216 @@
 $(document).ready(function(){
 
+  var APIaction = function(){
+    
+  };
 
+  APIaction.prototype.logOut = function(){
+    $.ajax({
+      type: 'DELETE',
+      url: '/sessions',
+      success: function(response){
+        alert("Logout successful. See you soon!")
```

Comment:
If it works, please remove the `alert` and consider using something like [bootstrap alerts](http://getbootstrap.com/components/#alerts)